config/sync/entity_print.settings.yml

```diff
@@ -1,8 +1,9 @@
 _core:
   default_config_hash: nF3sn8D7n0S0cxMK-kk9HmtudLA5iIm0EXJo7xJQQRY
 default_css: false
-force_download: true
+force_download: false
```

Comment:
This should not be changed. `force_download: false` should only be used during development.

Reply:
Changed